src/main/java/Dog.java

```diff
@@ -15,17 +16,18 @@ public Dog(String name, String owner){
         System.out.println("It's Owner is "+name);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)